app/(auth)/signup/page.tsx

```diff
@@ -0,0 +1,20 @@
+import SignUpForm from '@/components/shared/forms/SignUpForm';
+import Link from 'next/link';
+import React from 'react';
+
+function SignUp() {
+  return (
+    <div className="w-full h-full flex flex-col p-4 bg-green">
+      <div className="w-[300px] h-[120px] border-4 border-solid rounded-sm border-slate-800 flex justify-center items-center">
+        <p>Placeholder Image</p>
```

Comment:
Do we have a nice image already that we want to use here?

Reply:
still unsure :(